src/brevitas_examples/common/learned_round/learned_round_parser.py

```diff
@@ -0,0 +1,86 @@
+# Copyright (C) 2024, Advanced Micro Devices, Inc. All rights reserved.
+# SPDX-License-Identifier: BSD-3-Clause
+
+from typing import Any, Callable, Dict, Optional, Tuple, Type, Union
+import warnings
+
+import torch
+from torch.optim.lr_scheduler import LRScheduler
+from torch.optim.optimizer import Optimizer
+
+from brevitas.inject.enum import LearnedRoundImplType
+from brevitas.optim.sign_sgd import SignSGD
+from brevitas_examples.common.learned_round.learned_round_method import LearnedRound
+from brevitas_examples.common.learned_round.learned_round_method import LearnedRoundLoss
+from brevitas_examples.common.learned_round.learned_round_method import MSELoss
+from brevitas_examples.common.learned_round.learned_round_method import RegularisedMSELoss
+
+LEARNED_ROUND_MAP = {
+    "linear_round": LearnedRoundImplType.IDENTITY,
+    "hard_sigmoid_round": LearnedRoundImplType.HARD_SIGMOID,
+    "sigmoid_round": LearnedRoundImplType.SIGMOID,}
+LEARNED_ROUND_LOSS_MAP = {
+    "mse": MSELoss,
+    "regularised_mse": RegularisedMSELoss,}
+OPTIMIZER_MAP = {
+    "sign_sgd": SignSGD,}
+LR_SCHEDULER_MAP = {}
```

Comment:
This should not be empty, since we define one (LinearLR)

Reply:
That dictionary is only for custom LR Schedulers, default PyTorch ones are retrieved from torch.optim.lr_scheduler by matching the string passed by the user.